code/game/world.dm

```diff
@@ -353,30 +339,25 @@ GLOBAL_PROTECT(tracy_init_reason)
 		if(do_hard_reboot)
 			log_world("World hard rebooted at [time_stamp()]")
 			shutdown_logging() // See comment below.
-			shutdown_byond_tracy()
-			auxcleanup()
+			del(Tracy)
+			del(Debugger)
 			TgsEndProcess()
 			return ..()
 
 	log_world("World rebooted at [time_stamp()]")
 
 	shutdown_logging() // Past this point, no logging procs can be used, at risk of data loss.
-	shutdown_byond_tracy()
-	auxcleanup()
+	del(Tracy)
+	del(Debugger)
```

Comment:
Should use QDEL_NULL with QDEL_HINT_HARDDEL_NOW for convention

Why do these need to be del'd though?

Reply:
fair, mostly just thought "they cleanup/destroy on deletion" made sense rather than just calling `Tracy.shutdown()` or whatever - but if you would prefer the latter, I'd be fine with changing it.

and yeah I could just do `QDEL_HINT_HARDDEL_NOW`

Reply:
Do they need to harddel though? Seems like wasted cycles. Could just regular qdel

Reply:
is, uh, is SSgarbage gonna even be running here?

Reply:
we only really need destroy called in this case, doesn't have to actually be GC'd/harddeled

Reply:
fair